products/pan-cortex-xdr.md

```diff
@@ -1,15 +1,26 @@
 ---
 title: Palo Alto Networks Cortex XDR agent
 category: app
-permalink: /cortexxdr
+permalink: /cortex-xdr
 releasePolicyLink: https://www.paloaltonetworks.com/services/support/end-of-life-announcements/end-of-life-summary
 changelogTemplate: https://docs.paloaltonetworks.com/cortex/cortex-xdr/{{"__RELEASE_CYCLE__" | remove:'-' | replace:'.','-'}}/cortex-xdr-agent-release-notes/cortex-xdr-agent-release-information
 iconSlug: NA
 activeSupportColumn: false
 releaseColumn: false
 releaseDateColumn: true
-eolColumn: End-of-life Date
+eolColumn: Support Status
+alternate_urls:
+-   /pan-xdr
+-   /cortexxdr
+-   /xdr
+-   /pan-cortex-xdr
 releases:
+# EOL date for 7.9 is not yet published
```

Comment:
It is now ;) : https://www.paloaltonetworks.com/services/support/end-of-life-announcements/end-of-life-summary.

> 7.9 (Cortex XDR agent) | December 4, 2022 | September 11, 2023